.github/workflows/deploy.yml

```
@@ -27,11 +27,22 @@ jobs:
           PIP_ONLY_BINARY: netCDF4  # prefer wheel, fail fast if none
         run: |
           python -m pip install --upgrade pip
-          pip install -r requirements.txt
-          pip install sphinx pydata-sphinx-theme sphinx-design
-
-      - name: Install Abil
-        run: pip install .
+          pip install -r requirements.txt  # Ensure your requirements.txt includes Sphinx
+          pip install matplotlib
+          pip install seaborn
+          pip install Cartopy
+          pip install -e . #abil install
+          pip install sphinx
+          pip install pydata-sphinx-theme
+          pip install sphinx-design
+          pip install sphinxcontrib-bibtex
```

```suggestion
          pip install -e . #abil install
          pip install -r documentation_requirements.txt
```

but double check the documentation_requirements.txt is the right path

---

Got it, wasn't sure how to set that up properly, thanks :)